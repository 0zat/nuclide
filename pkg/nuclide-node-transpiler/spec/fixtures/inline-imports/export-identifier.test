import Foo from './foo';
import {Bar} from './bar';

export {Foo as foo};
export {Foo, Bar};
