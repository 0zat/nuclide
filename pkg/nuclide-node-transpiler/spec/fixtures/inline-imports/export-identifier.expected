Object.defineProperty(exports, '__esModule', {
  value: true
});

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { 'default': obj }; }

var _foo;

function _load_foo() {
  return _foo = _interopRequireDefault(require('./foo'));
}

var _bar;

function _load_bar() {
  return _bar = require('./bar');
}

exports.foo = (_foo || _load_foo()).default;
exports.Foo = (_foo || _load_foo()).default;
exports.Bar = (_bar || _load_bar()).Bar;
